### DISCUSSION TITLE
Customizing Aircraft Movement in BlueSky Simulator Using Historical Data

### DISCUSSION THREAD

**Discussion by HomaizaSaud** [TUDelft-CNS-ATM/bluesky#511] [Q&A]
I'm sorry this may seem a bit obvious, but I still wanted to confirm. What is the reason the aircraft moves on it's own when you create one? And why does the altitude change as well? Does this have to do with the autopilot logic? If I want to prevent this, do I modify the autopilot script? 
I want the positions of aircraft to update based on the data I provide including the time it should move at since I am using historical data. 
Thank you and any help would be appreciated.

**Comment by jooste** [comment]
Hi @HomaizaSaud,

Unless you create an aircraft with zero speed on the ground it will immediately start moving. Some states may start changing immediately as well if they are initialised outside the performance envelope of the aircraft. It would help if you provided a minimum working example of a create command that leads to the behaviour you describe.

**Reply by HomaizaSaud** [reply]
Hello there,

Thank you for your response. What I meant is that I have data from ASTERIX files where the information for a specific aircraft is only updated once around every 2.5-3.5 seconds. How can I make it so that the aircraft positions are updated based on this data within the simulator rather than updating automatically? Is there a way to achieve this?
